frontend/public/components/image-stream.tsx

```diff
@@ -107,6 +109,38 @@ const ImageStreamTagsRow: React.SFC<ImageStreamTagsRowProps> = ({imageStream, sp
   </div>;
 };
 
+const ImageStreamManipulationHelp: React.FC<ImageStreamManipulationHelpProps> = ({imageStream}) => {
```

Comment:
Can you change this to `ExampleDockerCommandPopover`?

Reply:
Not sure why, but I swear that I've made this change.